test/map.spec.js

```diff
@@ -1,28 +1,94 @@
 import test from 'tape';
-import * as map from '../src/helpers/map';
-import { tileTypes, teams } from '../src/constants';
-import { mockMe } from '../src/helpers/player';
-
-test('test isTraversable returns correct value', t => {
-  t.true(map.isTraversable(2)); // Regular floor
-  t.true(map.isTraversable(3.1)); // taken red flag
-  t.true(map.isTraversable(9)); // inactive gate
-  t.true(map.isTraversable(17)); // red endzone
-  t.false(map.isTraversable(0)); // Blank space
-  t.false(map.isTraversable(1)); // square wall
-  t.false(map.isTraversable(7)); // spike
+import {
+  init2dArray,
+  isTraversable,
+  getNonTraversableObjectRadius,
+  fillGridWithSubgrid,
+  addBufferTo2dArray,
+  getSubarrayFrom2dArray,
+  getTileTraversabilityInCells,
+  getMapTraversabilityInCells,
+  multiplyCorrespondingElementsAndSum,
+  convolve,
+  __RewireAPI__ as MapRewireAPI,
+} from '../src/helpers/map';
+import { tileTypes } from '../src/constants';
+
+
+test('init2dArray: returns correctly with varying inputs', t => {
```

Comment:
I'm not sure I like "varying inputs", but I think its okay. I'm not sure its worth it to split up the tests for each input and then naming the tests